usage.go

```diff
@@ -95,6 +95,13 @@ func (p *Parser) WriteHelp(w io.Writer) {
 				}
 				fmt.Fprint(w, spec.help)
 			}
+			// Check if spec.dest is zero value or not
+			// If it isn't a default value have been added
+			v := spec.dest
+			z := reflect.Zero(v.Type())
+			if v.Interface() != z.Interface() {
+				fmt.Fprintf(w, " [default: %s]", v)
```

Comment:
For non-strings, I believe this will end up printing something like `%!s(reflect.Value=1)`. Can you use `%q` instead?